assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -408,10 +408,20 @@ const baseResolvers = {
 		// If we have inactive dependencies, there's no need to check if we can
 		// activate the module until the dependencies have been activated.
 		if ( inactiveModules.length ) {
-			const formatter = new Intl.ListFormat( getLocale(), { style: 'long', type: 'conjunction' } );
-
 			/* translators: Error message text. 1: A flattened list of module names. 2: A module name. */
-			const errorMessage = sprintf( __( 'You need to set up %1$s to gain access to %2$s.', 'google-site-kit' ), formatter.format( inactiveModules ), module.name );
+			const messageTemplate = __( 'You need to set up %1$s to gain access to %2$s.', 'google-site-kit' );
+
+			let errorMessage;
+
+			// Not all browsers support Intl.Listformat per
+			// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat#Browser_compatibility
+			// We've seen that the built versions don't polyfill for the unsupported browsers (iOS/safari) so we provide a fallback.
+			if ( Intl.Listformat ) {
+				const formatter = new Intl.ListFormat( getLocale(), { style: 'long', type: 'conjunction' } );
+				errorMessage = sprintf( messageTemplate, formatter.format( inactiveModules ), module.name );
+			} else {
+				errorMessage = sprintf( messageTemplate, inactiveModules.join( ', ' ), module.name );
```

Comment:
@ivankruchkoff please use a translatable list separator like we do in a few other places:
https://github.com/google/site-kit-wp/blob/e081e1327a497b7e158a2bd7f2af12a9b0d6a506/assets/js/components/notifications/UnsatisfiedScopesAlert.js#L91-L92

It would probably be good to extract this to a `listFormat` utility like we have for `numberFormat` as well (just the list formatting part of course 😄 )